README.md

```diff
@@ -31,7 +31,7 @@ CSI driver.
 
 ### Note
 
-* Master is for CSI v0.4.0. Please see the branches for other CSI releases.
+* Master is for CSI v1.2.0. Please see the branches for other CSI releases.
```

Comment:
The golang note is also out of date.

What do you think about having a compatibility matrix similar to what the other sidecar repos have?

Reply:
@msau42 thought about the `golang` , but it mention about minimum supported version ( here `1.9+`), so is it correct to mention that we need `1.12`+ for master ? 

Regarding the compatibility matrix that's a good idea :+1: . However I have to fetch the details somehow. Its also possible to have a followup PR for compatibility matrix.

Reply:
I'm not sure what the min version of golang is. Our CI uses go1.13. @pohly do you have any thoughts here?

Reply:
I don't know what the oldest version of Go is that the code still builds with.

I suppose a correct statement would be "building has been tested with the Go version specified in release-tools/travis.yml"...

Reply:
Yeah, that would be better, confirming the golang compatibility will be a tediuous task unless we have building infra with all the different versions of `go`. I have updated the PR to include above statement. PTAL @pohly @msau42